campaignion_donation_amount/campaignion_donation_amount.module

```diff
@@ -19,6 +19,10 @@ function campaignion_donation_amount_webform_component_render_alter(array &$elem
     $element['#element_validate'][] = 'campaignion_donation_amount_validate';
     // Always show the select_or_other textfield.
     $element['#attributes']['data-select-or-other-hide'] = 0;
+    if ($symbol = variable_get_value('campaignion_donation_amount_currency_symbol')) {
+      $element['#other_title'] = $symbol;
+      $element['#other_title_display'] = 'before';
+    }
```

Comment:
The amount component does not necessarily have to be of type `select_or_other` – do we want to add a "#other_title" to a text field or plain radios?

Reply:
Unused form-API `#properties` don’t hurt in any way. What’s better?

1. Set it unconditionally and have it ignored by other `#type`s
2. Add an additional check for `$element['#type'] == 'select_or_other'`?

Reply:
I'd prefer 2., then it's clearer in which cases we expect the property (and js attribute) to be used.

Reply:
Updated with the latest commit.

Reply:
Perfect, thanks!